sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/implementation/ServiceBusAsyncConsumer.java

```diff
@@ -20,13 +21,15 @@
     private final ServiceBusReceiveLinkProcessor amqpReceiveLinkProcessor;
     private final MessageSerializer messageSerializer;
     private final ServiceBusMessageProcessor processor;
+    private final Scheduler scheduler;
 
     public ServiceBusAsyncConsumer(ServiceBusReceiveLinkProcessor amqpReceiveLinkProcessor,
-        MessageSerializer messageSerializer, boolean isAutoComplete,
-        Function<ServiceBusReceivedMessage, Mono<Void>> onComplete,
-        Function<ServiceBusReceivedMessage, Mono<Void>> onAbandon) {
+                                   MessageSerializer messageSerializer, boolean isAutoComplete,
```

Comment:
Undo this reformat. The parameters are all mushed against the right margin. I have it set in intellij not to align it.